projects/igniteui-angular/src/lib/splitter/splitter-bar/splitter-bar.component.ts

```diff
@@ -38,7 +36,18 @@ export class IgxSplitBarComponent {
      * @internal
      */
     @HostBinding('attr.tabindex')
-    public tabindex = 0;
+    public get tabindex() {
+        return this.resizeDisallowed ? null : 0;
+    }
+
+    /**
+     * @hidden
+     * @internal
+     */
+    @HostBinding('attr.aria-orientation')
+    public get orientation() {
+        return this.type === SplitterType.Horizontal ? 'vertical' : 'horizontal';
```

Comment:
Actually I think those march - i.e. a horizontal splitter is still described with horizontal bar (separator)

Reply:
That means it this may even be directly bound to type itself (assuming enum values match 100%)